docs/contributing/Target Framework Strategy.md

```diff
@@ -9,6 +9,7 @@ The roslyn repository produces components for a number of different products tha
 - Source build: requires us to ship `$(NetCurrent)` and `$(NetPrevious)` in workspaces and below (presently `net9.0` and `net8.0` respectively)
 - Visual Studio: requires us to ship `net472` for base IDE components and `$(NetVisualStudio)` (presently `net8.0`) for private runtime components.
 - Visual Studio Code: expects us to ship against the same runtime as DevKit (presently `net7.0`) to avoid two runtime downloads.
+- MSBuildWorkspace: requires to ship a process that must be usable on the lowest supported SDK (presently `net6.0`)
```

Comment:
I guess the expiration date for this Nov 12th 2024?
https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#lifecycle

Reply:
Well, depends on whether we support officially unsupported things. Which is often case in the tooling world...

Reply:
This reminds me that I need to pin an O# announcement about dropping .NET 6/7 support in November.